prow/plugins/wip/wip-label_test.go

```diff
@@ -103,14 +97,6 @@ func TestWipLabel(t *testing.T) {
 		} else if len(fc.LabelsRemoved) > 0 {
 			t.Errorf("For case %s, expected to not remove %q label but removed: %v", tc.name, label, fc.LabelsRemoved)
 		}
-
-		if tc.shouldComment {
-			if len(fc.IssueCommentsAdded) != 1 || fc.IssueCommentsAdded[0] != fmt.Sprintf("%s/%s#%d:%s", org, repo, number, body) {
-				t.Errorf("For case %s: expected to add comment but instead added: %v", tc.name, fc.IssueCommentsAdded)
-			}
-		} else if len(fc.IssueCommentsAdded) > 0 {
-			t.Errorf("For case %s, expected to not add comment but added: %v", tc.name, fc.IssueCommentsAdded)
```

Comment:
Can we keep this `if` condition?

Reply:
I don't think it's necessary to keep this.